ui/src/views/tools/ManageInstances.vue

```diff
@@ -1408,11 +1408,31 @@ export default {
         })
       }
     },
+    async fetchMoreVms (obj) {
+      var params = obj.params
+      params.token = obj.response.token
+      api('listVmwareDcVms', params).then(json => {
+        const obj = {
+          params: params,
+          response: json.listvmwaredcvmsresponse
+        }
+        this.unmanagedInstances.append(obj.response.unmanagedinstance)
+        this.checkForMoreVms(obj)
+      })
+    },
     onListUnmanagedInstancesFromVmware (obj) {
       this.selectedVmwareVcenter = obj.params
       this.unmanagedInstances = obj.response.unmanagedinstance
       this.itemCount.unmanaged = obj.response.count
-      this.unmanagedInstancesLoading = false
+      this.checkForMoreVms(obj)
+    },
+    checkForMoreVms (obj) {
```

Comment:
Is it better to let user control loading more objects or not? Currently, if there are 1000 VMware VMs and user opens this view, user's intended VM is returned in the first request itself, will it continue calling API for more results?

Reply:
I am still thinking on how to make this the best user experience, so far I just want the background loading to work while the use no longer needs to wait.